MLDataR-post/MLDataR-post.Rmd

```diff
@@ -0,0 +1,53 @@
+---
+title: "MLDataR - A Package for Exploring ML in Healthcare"
```

Comment:
Since the package has expanded beyond healthcare, what do you think of something like "MLDataR - A Package for Exploring Real-World Machine Learning Data" or something like that?